doc/contributing/releases.md

```diff
@@ -1493,5 +1482,4 @@ Typical resolution: sign the release again.
 [`create-release-proposal`]: https://github.com/nodejs/node/actions/workflows/create-release-proposal.yml
 [build-infra team]: https://github.com/orgs/nodejs/teams/build-infra
 [expected assets]: https://github.com/nodejs/build/tree/HEAD/ansible/www-standalone/tools/promote/expected_assets
-[nodejs.org release-post.js script]: https://github.com/nodejs/nodejs.org/blob/HEAD/apps/site/scripts/release-post/index.mjs
```

Comment:
non-blocking nit: this script is still mentioned as a fallback - perhaps it should remain?